bundles/org.openhab.ui/web/src/pages/settings/things/channel/channel-edit.vue

```diff
@@ -11,7 +11,7 @@
     <f7-block class="block-narrow">
       <f7-col v-if="channel">
         <f7-block-title>Channel</f7-block-title>
-        <channel-general-settings :channel="channel" :createMode="false" :ready="ready" />
+        <channel-general-settings :channel="channel" :channelType="channelType" :createMode="false" ready="true" />
```

Comment:
I didn't notice it before but I think it's quite weird to have a `ready` prop - in my mind it's a component-local flag and should remain so, normally you would only need to switch from a "loading" state to a "ready" state in your component, if you have data to load, and replace the UI with `v-if`/`v-else`; not have the subcomponent handle the "not ready" state.

Reply:
You are right, I will implement this.